sdk/logs/src/main/java/io/opentelemetry/sdk/logs/ExtendedSdkLogger.java

```diff
@@ -13,19 +13,11 @@
 /** SDK implementation of {@link ExtendedLogger}. */
 final class ExtendedSdkLogger extends SdkLogger implements ExtendedLogger {
 
-  private final boolean loggerEnabled;
-
   ExtendedSdkLogger(
       LoggerSharedState loggerSharedState,
       InstrumentationScopeInfo instrumentationScopeInfo,
       LoggerConfig loggerConfig) {
     super(loggerSharedState, instrumentationScopeInfo, loggerConfig);
-    this.loggerEnabled = loggerConfig.isEnabled();
-  }
-
-  @Override
-  public boolean isEnabled() {
```

Comment:
Let's keep this around.

Reply:
What for? `isEnabled` is inherited from `SdkLogger`, am I missing something?

Reply:
Its a minor thing: ExtendedSdkLogger implemented ExtendedLogger, which is where the isEnabled() API is defined.

If we only define it in SdkLogger, we can't keep the `@Override` annotation.

Reply:
Got it, fixed in 4d3871c84b10775d4f69ce123b210392fecaf4fa